specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/examples/ManagedClustersCreate_PPG.json

```diff
@@ -0,0 +1,236 @@
+{
+  "parameters": {
+    "api-version": "2020-12-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "parameters": {
+      "location": "location1",
+      "tags": {
+        "tier": "production",
+        "archv2": ""
+      },
+      "sku": {
+        "name": "Basic",
+        "tier": "Free"
+      },
+      "properties": {
+        "kubernetesVersion": "",
+        "dnsPrefix": "dnsprefix1",
+        "agentPoolProfiles": [
+          {
+            "name": "nodepool1",
+            "count": 3,
+            "vmSize": "Standard_DS2_v2",
+            "osType": "Linux",
+            "type": "VirtualMachineScaleSets",
+            "enableNodePublicIP": true,
```

Comment:
there is a new field(`EnableEncryptionAtHost    *bool`) added per agent pool in 20201201, could you also add that field? thanks
```
"enableEncryptionAtHost": false,
```